api/v1alpha2/linodemachine_webhook.go

```diff
@@ -113,8 +113,14 @@ func (r *LinodeMachine) validateLinodeMachineSpec(ctx context.Context, client Li
 	// TODO: instrument with tracing, might need refactor to preserve readibility
 	var errs field.ErrorList
 
-	if err := validateRegion(ctx, client, r.Spec.Region, field.NewPath("spec").Child("region")); err != nil {
-		errs = append(errs, err)
+	if r.Spec.DiskEncryption == "enabled" {
```

Comment:
given LDE will be enabled by default platform wide I think we can remove this check now?

Reply:
Is LDE available in all DCs yet? If not we should leave this in

Reply:
since LDE is enabled by default, the use case for this flag is not actually to turn LDE on, but rather to turn it off. I think that would make this check a noop in almost all situations.